Dockerfile

```diff
@@ -15,6 +17,11 @@ LABEL io.k8s.description="ASP.NET 5.0" \
 
 RUN printf "deb http://ftp.us.debian.org/debian jessie main\n" >> /etc/apt/sources.list
```

Comment:
@alberttwong @jawnsy do you want me to collapse it all in this PR or fix my changes, merge and then combine?